src/components/Modal.js

```diff
@@ -10,7 +10,7 @@ import * as Style from 'src/libs/style'
 import * as Utils from 'src/libs/utils'
 
 
-const styles = {
+export const styles = {
```

Comment:
nit: This is my only concern in this change. Exporting the style makes it a bit more difficult to change in the future if it is used in a lot of places. But I don't have a better solution in mind right now.

Reply:
To add the switch to the modal footer, I had to set `showButtons: false` on the Modal and render replacement buttons. 
I added this export thinking that, this way, the footer here would stay up to date with any changes to the base modal footer styles. I can remove this export and copy and paste the styles, but that has its own drawbacks.